bundles/1.1.4/manifests/cloud-resource-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,380 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "BlobStorage",
+          "metadata": {
+            "labels": {
+              "productName": "ProductName"
+            },
+            "name": "example-blobstorage"
+          },
+          "spec": {
+            "applyImmediately": false,
+            "maintenanceWindow": false,
+            "secretRef": {
+              "name": "example-blobstorage-sec"
+            },
+            "tier": "development",
+            "type": "REPLACE_ME"
+          }
+        },
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "Postgres",
+          "metadata": {
+            "labels": {
+              "productName": "productName"
+            },
+            "name": "example-postgres"
+          },
+          "spec": {
+            "maintenanceWindow": false,
+            "secretRef": {
+              "name": "example-postgres-sec"
+            },
+            "tier": "development",
+            "type": "REPLACE_ME"
+          }
+        },
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "PostgresSnapshot",
+          "metadata": {
+            "name": "example-postgressnapshot"
+          },
+          "spec": {
+            "resourceName": "REPLACE_ME"
+          }
+        },
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "Redis",
+          "metadata": {
+            "labels": {
+              "productName": "productName"
+            },
+            "name": "REPLACE_ME"
+          },
+          "spec": {
+            "applyImmediately": false,
+            "maintenanceWindow": false,
+            "secretRef": {
+              "name": "REPLACE_ME-sec"
+            },
+            "size": "REPLACE_ME",
+            "tier": "development",
+            "type": "REPLACE_ME"
+          }
+        },
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "RedisSnapshot",
+          "metadata": {
+            "name": "example-redissnapshot"
+          },
+          "spec": {
+            "resourceName": "REPLACE_ME"
+          }
+        }
+      ]
+    capabilities: Basic Install
+    containerImage: quay.io/integreatly/cloud-resource-operator:v1.1.4
```

Comment:
@valerymo that image doesnt seem to exist, I forgot the release process of CRO but is that expected?

Reply:
@MStokluska , thank you for comment. 
- I worked according to Release preparing notes in CRO README.   No notes for new index image preparing there. In Description I wrote that Local installation was verified.  
- Similar task for CRO 1.1.3 - yes, it includes images build with Pipeline. https://github.com/integr8ly/cloud-resource-operator/pull/723
- So -  I will  build the index image similar as it was for 1.1.3, and will test it.  
Thank you